examples/build_file_generation/BUILD

```
@@ -0,0 +1,37 @@
+load("@bazel_gazelle//:def.bzl", "gazelle")
+load("@rules_python//gazelle:def.bzl", "GAZELLE_PYTHON_RUNTIME_DEPS")
+load("@rules_python//gazelle/manifest:defs.bzl", "gazelle_python_manifest")
+load("@rules_python//python:defs.bzl", "py_library")
+
+# Gazelle python extension needs a manifest file mapping from
+# an import to the installed package that provides it.
+# This macro produces two targets:
+# - //:gazelle_python_manifest.update can be used with `bazel run`
+#   to recalculate the manifest
+# - //:gazelle_python_manifest.test is a test target ensuring that
+#   the manifest doesn't need to be updated
+gazelle_python_manifest(
+    name = "gazelle_python_manifest",
+    modules_mapping = "@modules_mapping//:modules_mapping.json",
```

```suggestion
    modules_mapping = "@modules_map//:modules_mapping.json",
```

Noticed from hitting `ERROR: /Users/jonathon/work/rules_python/examples/build_file_generation/BUILD:13:24: no such package '@modules_mapping//': The repository '@modules_mapping' could not be resolved and referenced by '//:gazelle_python_manifest.update'` when running this example.